src/wallet/walletdb.h

```diff
@@ -46,9 +46,12 @@ class CHDChain
 {
 public:
     uint32_t nExternalChainCounter;
+    uint32_t nInternalChainCounter;
     CKeyID masterKeyID; //!< master key hash160
 
-    static const int CURRENT_VERSION = 1;
+    static const int VERSION_HD_BASE        = 1;
```

Comment:
`VERSION_HD_BASE` is only there to identify versions that do not support HD SPLIT (first HD version introduced in 0.13).